src/amo/views/sidebarView.ts

```diff
@@ -33,6 +33,22 @@ export class AssayTreeDataProvider
             title: "Download and Extract Addon",
           }
         ),
+        new AssayTreeItem(
+          "Enter API Key",
+          vscode.TreeItemCollapsibleState.None,
+          {
+            command: "assay.getApiKey",
+            title: "Enter AMO API Key",
+          }
+        ),
+        new AssayTreeItem(
+          "Enter API Secret",
+          vscode.TreeItemCollapsibleState.None,
+          {
+            command: "assay.getSecret",
+            title: "Enter AMO API Secret",
+          }
+        ),
```

Comment:
Having the option to set the api key/secret in the sidebar seems unnecessary when it's a one-time thing the user will need to set - can it be defined in the settings page instead?